CentralComputing/Sensors/Accelerometer.cpp

```diff
@@ -15,10 +15,11 @@ void Accelerometer::reset() {
 	//TODO: implement resetting
 }
 
-void Accelerometer::update() {
+void Accelerometer::update(Spi * spi) {
 
 	switch(simulation) {
```

Comment:
Would this be better as a `#ifdef` ?  This applies to all of the switch() statements inside of the sensor classes